spec/models/bookable_slot_spec.rb

```diff
@@ -79,13 +79,14 @@ def result
         )
       end
 
-      context 'one guider has a bookable slot obscured by an appointment' do
+      context 'one guider has a bookable slot obscured by a non cancelled appointment' do
         it 'excludes the slot' do
           create(
             :appointment,
             guider: User.guiders.first,
             start_at: make_time(10, 30),
-            end_at: make_time(11, 30)
+            end_at: make_time(11, 30),
+            status: Appointment.statuses['pending']
```

Comment:
Pretty sure you can just pass the statuses as symbols and they get coerced correctly.